### DISCUSSION TITLE
[Invalid] Why you sell this project to microsoft store? they just broke it!

### DISCUSSION THREAD

**Discussion by Rafriell** [KinectToVR/Amethyst-Releases#6] [General]
Stabile -1 error connection since then.

**Comment by meruune** [comment]
LMAO this definitely belongs to the starboard. 

Most errors are exclusive to certain circumstances, and not to the Microsoft Store. Please learn to read, some basic common sense, (maybe grammar, too?) and reach to us on Discord. Remember to describe your error properly, and make sure to read the docs first - maybe the error you’re encountering has already got a dedicated fix. 

That said, thanks for your input, and sincerely, f’ck you.
